test/streaming_engine_unit.js

```diff
@@ -1950,6 +1952,7 @@ describe('StreamingEngine', function() {
    * Initializes or switches to the given period.
    *
    * @param {shakaExtern.Period} period
+   * @return {Object}
```

Comment:
I changed to ```@return {{audio: string, video: string, text: string}}```.

As you know the shaka code better than I can you suggest a tighter type?